setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='resigner',
-    version='0.3.2',
+    version='0.3.4',
```

Comment:
skipping 0.3.3 since it has already been consumed through a git tag here: https://github.com/mediapredict/resigner/releases